test/extensions/filters/network/common/fuzz/uber_per_writefilter.cc

```diff
@@ -30,6 +32,31 @@ std::vector<absl::string_view> UberWriteFilterFuzzer::filterNames() {
   return filter_names;
 }
 
+void UberWriteFilterFuzzer::checkInvalidInputForFuzzer(const std::string& filter_name,
+                                                       Protobuf::Message* config_message) {
+  // System calls such as reading files are prohibited in this fuzzer. Some inputs that crash the
+  // mock/fake objects are also prohibited. We could also avoid fuzzing some unfinished features by
+  // checking them here. For now there is only one filter {MongoProxy} on which we have a
+  // constraint.
+  const std::string name = Extensions::NetworkFilters::Common::FilterNameUtil::canonicalFilterName(
+      std::string(filter_name));
+  if (filter_name == NetworkFilterNames::get().MongoProxy) {
+    envoy::extensions::filters::network::mongo_proxy::v3::MongoProxy& config =
+        dynamic_cast<envoy::extensions::filters::network::mongo_proxy::v3::MongoProxy&>(
+            *config_message);
+    if (config.has_delay() && config.mutable_delay()->has_header_delay()) {
```

Comment:
`mutable_delay()` allows you to change the value of the delay, but the code here only reads the value. 

Can you use `config.delay().has_header_delay()` instead?